content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md

```diff
@@ -0,0 +1,1359 @@
+title: darktable 2.6
+author: moy
+date: 2018-12-24
+lede: butterfly.jpg
+lede_author: <a href="https://matthieu-moy.fr/">Matthieu Moy</a>
+tags: announcement, darktable-release
+
+Following the tradition, darktable 2.6 was released for Christmas.
+2018 has been a year of renewal for darktable, with many major
+features introduced by recent contributors.
+
```

Comment:
Here, you could add:

`The announcement and release notes for this new release can be found here: [https://www.darktable.org/2018/12/darktable-260-released/](https://www.darktable.org/2018/12/darktable-260-released/).`